ncm-network/src/main/perl/network.pod

```diff
@@ -102,49 +134,52 @@ Setting nozeroconf to true stops an interface from being assigned an automatic a
 
 =head1 HWADDR
 
-Explicitly set the MAC address for the interfaces in the configuration files. The MAC address is taken from /hardware/cards/nic/eth[i]/hwaddr.
+Explicitly set the MAC address for the interfaces in the configuration files. The MAC address is taken from C<< /hardware/cards/nic/eth[i]/hwaddr >>.
+
+=over
 
-=item /system/network/set_hwaddr : boolean
+=item C<< /system/network/set_hwaddr : boolean >>
 
 Set the default behaviour for all interfaces. The component default is false.
 
-=item /system/network/interfaces/eth[i]/set_hwaddr : boolean
+=item C<< /system/network/interfaces/eth[i]/set_hwaddr : boolean >>
 
 Set the behaviour for interface eth[i]. This overrides the default setting.
 
+=back
 
 =head1 CHANNEL BONDING
 
-(see <kernel>/Documentation/networking/bonding.txt for more info on the driver options)
+(see C<< <kernel>/Documentation/networking/bonding.txt >> for more info on the driver options)
 
 To enable channel bonding with quattor using devices eth0 and eth1 to form bond0, proceed as follows:
 
-include pro_software_component_network;
-                                                                             
-"/system/network/interfaces/eth0/bootproto"="none";
-"/system/network/interfaces/eth0/master"="bond0";
+    include 'components/network/config';
+
+    "/system/network/interfaces/eth0/bootproto"="none";
+    "/system/network/interfaces/eth0/master"="bond0";
+
+    "/system/network/interfaces/eth1/bootproto"="none";
+    "/system/network/interfaces/eth1/master"="bond0";
 
-"/system/network/interfaces/eth1/bootproto"="none";
-"/system/network/interfaces/eth1/master"="bond0";
+    "/system/network/interfaces/bond0" = NETWORK_PARAMS;
```

Comment:
The change from cushioned = (with spaces) from "uncushioned =" (without spaces) is a bit jarring. Would it be possible to clean this up and use the same consistent style?